apps/site/lib/site_web/views/paragraph_view.ex

```diff
@@ -99,6 +100,26 @@ defmodule SiteWeb.CMS.ParagraphView do
     )
   end
 
+  def render_paragraph_content(%TripPlanWidget{} = paragraph, conn) do
```

Comment:
Have you considered just using the default render_paragraph_content clause, and then sending the values to the trip planner widget via template? These custom clauses were made because something about the paragraph needed to exist "above" the paragraph type template, but it seems here you're just extracting field values to send to a template.

`render_paragraph_content` -> paragraph_type_template -> render tripplanwidget w/content->field vals

Reply:
This is a good point, I will look into this!